### ISSUE TITLE
pochhammer test failure with mpmath==1.1.0

### ISSUE THREAD

**Issue by mtmiller** [cbm755/octsympy#918]
Since mpmath 1.1.0 came out last week, one test is reproducibly failing now

```
***** test
 % maple, matrix inputs
 A = [5.61467232547723663908 20.6144884613920190965];
 B = pochhammer ([0.9 0.8], [3.1 4.2]);
 assert (A, B, -eps);
!!!!! test failed
ASSERT errors for:  assert (A,B,-eps)

  Location  |  Observed  |  Expected  |  Reason
    (2)        20.6145      20.6145      Rel err 5.1702e-16 exceeds tol 2.2204e-16 by 3e-16
```

This is on x86-64, SymPy 1.3.

First [reported](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=916961) from Debian CI, and I was able to reproduce in a virtualenv where mpmath was updated to the newest release.

**Comment by mtmiller** [comment]
I should have mentioned, this test case is in `@double/pochhammer.m`, not in `@sym/pochhammer.m`.

**Comment by mtmiller** [comment]
PS - @cbm755 you might want to add a weekly cron build to your Travis settings to catch errors due to system changes when development is on a holiday.

**Comment by cbm755** [comment]
Thanks, and cron build done.

**Comment by mtmiller** [comment]
Fixed with ac8dc625eb7b, not sure why this isn't closed, commit tag typo?
